### ISSUE TITLE
Change localhost in Lets encrypte

### ISSUE THREAD

**Issue by inetactief** [AzuraCast/AzuraCast#614]
Hi,

My cert is using localhost now.  I try to renew to the domain what i use.

 I run ./docker.sh letsencrypt-create
When i use my dns name, hey say:
Domain name xxxxxxxx is not set up with LetsEncrypt yet

And hey say ERROR: No such service: letsencrypt 

How can i re entered in the previous step in letsencrypt?  Or remove and reinstall letsencrypt?

**Comment by SlvrEagle23** [comment]
@inetactief Have you recently updated your `docker-compose.yml` file? It should have a section under `services:` called `letsencrypt`. If not, you can copy the relevant section from the [sample file](https://github.com/AzuraCast/AzuraCast/blob/master/docker-compose.sample.yml#L51-L58).

Also don't forget to add the necessary [volumes](https://github.com/AzuraCast/AzuraCast/blob/master/docker-compose.sample.yml#L185-L187) to the bottom of the file.

**Comment by inetactief** [comment]
I have still the localhost cert.

**Comment by SlvrEagle23** [comment]
@inetactief Okay, but did you make sure the items I mentioned in my post above were done? That should at least fix the `no such service: letsencrypt` error.

**Comment by inetactief** [comment]
Yes it works now SSL and the good domain name... i must wait longer...;-)
